libs/python.json

```diff
@@ -46,6 +47,8 @@
   {"id":"py.django-elastic-transcoder","imports":["dj_elastictranscoder"],"name":"Django Elastic Transcoder","repo":"StreetVoice/django-elastic-transcoder","tags":["aws-transcoder"],"tech":["cloud"],"status":"model-ready"},
   {"id":"py.docker","imports":["docker"],"name":"Docker SDK for Python","repo":"docker/docker-py","tags":["docker"],"tech":["cloud"],"status":"awaiting-model"},
   {"id":"py.fabric","imports":["fabric"],"name":"fabric","repo":"fabric/fabric","tags":["deployment"],"tech":["cloud"],"status":"model-ready"},
+  {"id":"py.googleapis","imports":["google"],"name":"Google APIs","repo":"googleapis/googleapis","tags":["google-API-protocols"],"tech":["cloud"],"status":"awaiting-model"},
```

Comment:
We will have a problem with this library. I think we should exclude it for now. It's protocol buffers and not much else. If we train a classifier on this, it won't work.